src/execution/compiler/operator/hash_aggregation_translator.cpp

```diff
@@ -29,6 +29,21 @@ HashAggregationTranslator::HashAggregationTranslator(const planner::AggregatePla
   TERRIER_ASSERT(plan.GetAggregateStrategyType() == planner::AggregateStrategyType::HASH,
                  "Expected hash-based aggregation plan node");
   TERRIER_ASSERT(plan.GetChildrenSize() == 1, "Hash aggregations should only have one child");
+
+  for (size_t agg_term_idx = 0; agg_term_idx < plan.GetAggregateTerms().size(); agg_term_idx++) {
```

Comment:
Style/consistency nit: Is there a reason this was moved from below the GetGroupByTerms?

Reply:
This was so that `distinct_filters_` could be initialized, and we could then update the parallelism of `build_pipeline_`.